setup.py

```diff
@@ -7,6 +7,7 @@
 from setuptools import setup
 
 setup(
-    setup_requires=['pbr'],
+    setup_requires=['pbr', 'pytest-runner', 'wheel'],
+    tests_require=['pytest', ],
```

Comment:
isn't there a way to insert it into setup.cfg and keep setup.py as minimal as possible?

Also, useless comma. `tests_require=['pytest']` works.